extension/content.js

```diff
@@ -13,7 +13,7 @@ main()
 function main() {
     const videoId = getVideoId()
     if (videoId) {
-        Promise.all([fetchComments(videoId), fetchVideo(videoId)]).then(results => {
+        Promise.all([fetchAllComments(videoId), fetchVideo(videoId)]).then(results => {
```

Comment:
There are videos with thousands (even millions) of  comments. I'm not sure that we should fetch all of them. Probably 5 or 10 pages could be enough.

Reply:
Fixed in last commit, added a variable so it will be possible to dynamically change it in the future with new features